lib/dom-parser.js

```diff
@@ -248,83 +223,79 @@ DOMParser.prototype.parseFromString = function (source, mimeType) {
 	 */
 	this.mimeType = opt.mimeType || MIME_TYPE.XML_APPLICATION;
 
 	/**
-	 * The namespace to use to create an XML document.
-	 * For the following reasons this is required:
-	 * - The SAX API for `startDocument` doesn't offer any way to pass a namespace,
-	 *   since at that point there is no way for the parser to know what the default namespace from the document will be.
-	 * - When creating using `DOMImplementation.createDocument` it is required to pass a namespace,
-	 *   to determine the correct `Document.contentType`, which should match `this.mimeType`.
-	 * - When parsing an XML document with the `application/xhtml+xml` mimeType,
-	 *   the HTML namespace needs to be the default namespace.
+	 * The namespace to use to create an XML document. For the following reasons this is required:
+	 *
+	 * - The SAX API for `startDocument` doesn't offer any way to pass a namespace, since at that point there is no way for the
+	 * parser to know what the default namespace from the document will be.
+	 * - When creating using `DOMImplementation.createDocument` it is required to pass a namespace, to determine the correct
+	 * `Document.contentType`, which should match `this.mimeType`.
+	 * - When parsing an XML document with the `application/xhtml+xml` mimeType, the HTML namespace needs to be the default
+	 * namespace.
 	 *
-	 * @type {string|null}
+	 * @type {string | null}
+	 * @access private
 	 * @readonly
-	 * @private
 	 */
 	this.defaultNamespace = opt.defaultNamespace || null;
 
 	/**
-	 * @private
 	 * @type {boolean}
+	 * @access private
 	 */
 	this.cdata = false;
 
 	/**
-	 * The last `Element` that was created by `startElement`.
-	 * `endElement` sets it to the `currentElement.parentNode`.
+	 * The last `Element` that was created by `startElement`. `endElement` sets it to the `currentElement.parentNode`.
 	 *
 	 * Note: The sax parser currently sets it to white space text nodes between tags.
 	 *
 	 * @type {Element | Node | undefined}
-	 * @private
+	 * @access private
 	 */
 	this.currentElement = undefined;
 
 	/**
-	 * The Document that is created as part of `startDocument`,
-	 * and returned by `DOMParser.parseFromString`.
+	 * The Document that is created as part of `startDocument`, and returned by `DOMParser.parseFromString`.
 	 *
 	 * @type {Document | undefined}
 	 * @readonly
 	 */
 	this.doc = undefined;
 
 	/**
-	 * The locator is stored as part of setDocumentLocator.
-	 * It is controlled and mutated by the SAX parser
-	 * to store the current parsing position.
-	 * It is used by DOMHandler to set `columnNumber` and `lineNumber`
-	 * on the DOM nodes.
+	 * The locator is stored as part of setDocumentLocator. It is controlled and mutated by the SAX parser to store the current
+	 * parsing position. It is used by DOMHandler to set `columnNumber` and `lineNumber` on the DOM nodes.
 	 *
 	 * @type {Readonly<Locator> | undefined}
-	 * @readonly (the sax parser currently sometimes set's it)
-	 * @private
+	 * @access private
```

Comment:
I don't see any mention of the `@access` tag in the typescript docs: https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#property-modifiers

Is it possible to turn off the conversion of `@private` to `@access private`?

Reply:
Just in case, the plugin is based on JSDoc, not the JSDoc support on TSDoc.

You can use [`jsdocAllowAccessTag `](https://github.com/homer0/packages/tree/main/packages/public/prettier-plugin-jsdoc#allow-access-tag) to control the access tag.

Reply:
There is. [`jsdocEnforceAccessTag`](https://github.com/homer0/packages/blob/main/packages/public/prettier-plugin-jsdoc/README.md#enfore-the-use-of-the-access-tag)